### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,11 @@
 [travis-url]: https://travis-ci.org/tox-rs/ws-tox
 
 This server is used to connect JS and tox through WebSocket to create BUI.
+
+## Usage
+
+First you need Rust, libtoxcore and openssl installed. Then you can run ws-tox with
```
Comment (kpp) [inline README.md:12]: why openssl? Where is libsodium?

Comment (suhr) [reply README.md:12]: Libtoxcore contains libsodium and openssl is required for the websocket library.

Comment (kpp) [reply README.md:12]: No if you compile it from source

Comment (suhr) [reply README.md:12]: Sure, but we are not going to list all the dependencies of our dependencies, are we?

**Pull request merged**